app/javascript/context/subreducers/ApplicationDetailsSubreducer.js

```diff
@@ -0,0 +1,157 @@
+import { cloneDeep, findIndex } from 'lodash'
+
+import LEASE_STATES from 'context/actionCreators/application_details/leaseSectionStates'
+
+import ACTIONS from '../actions'
+
+export const getEmptyStatusModalState = () => ({
+  alertMsg: null,
+  header: null,
+  isOpen: false,
+  loading: false,
+  showAlert: false,
+  status: null,
+  substatus: null,
+  submitButton: null,
+  isInAddCommentMode: false
+})
+
+export const getEmptyApplicationDetailsState = () => ({
+  shortform: {
+    application: null,
+    fileBaseUrl: null
+  },
+  supplemental: {
+    application: null,
+    confirmedPreferencesFailed: false,
+    fileBaseUrl: null,
+    leaseSectionState: null,
+    leaveConfirmationModalOpen: false,
+    listing: null,
+    listingAmiCharts: null,
+    loading: false,
+    rentalAssistances: null,
+    statusHistory: null,
+    units: null,
+    applicationMembers: [],
+    statusModal: getEmptyStatusModalState()
+  }
+})
+
+const setSupplementalOverrides = (state, overrides = {}) => ({
+  ...state,
+  supplemental: {
+    ...state.supplemental,
+    ...overrides,
+    applicationMembers: [
+      state.supplemental.application?.applicant,
+      ...(state.supplemental.application?.household_members || [])
+    ].filter((v) => !!v),
+    statusModal: {
+      ...state.supplemental.statusModal,
+      ...overrides.statusModal
+    }
+  }
+})
+
+const APPLICATION_DETAILS_ACTIONS = {
+  [ACTIONS.LEFT_APPLICATION_SCOPE]: (_, __) => getEmptyApplicationDetailsState(),
```

Comment:
If you have thoughts on how to make this easier to read I'm open to suggestions!